frontend/src/app/types/user/NewUser.ts

```diff
@@ -0,0 +1,6 @@
+export interface NewUser {
```

Comment:
file name starts with upper N letter

also, use kebab-case for file names similar to app-routing.module.ts

here is actually an angular coding guidelines, it may be usefull
https://angular.io/guide/styleguide#avoid-aliasing-inputs-and-outputs

Reply:
Fixed